ra_aid/database/repositories/trajectory_repository.py

```
@@ -10,6 +10,7 @@
 import contextvars
 import json
 import logging
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)